client/src/components/MosaicLayout.css

```diff
@@ -38,6 +49,10 @@
 	z-index: 100;
 }
 
+.bp3-navbar {
+  margin-bottom: 18px;
```

Comment:
Can we replace `margin-bottom: 18px;` with `margin: 8px;` ?
I find it cleaner and less distracting if the mosaic boxes are not "nested" under the top bar.
If I simply do `margin: 8px;` then the right side is out of whack, I have not had time to look why...

Reply:
Yes agreed, was a bit more work to get that fixed as well. But it should work like a charm now :)